samples/generated_samples/bigquery_generated_bigquery_v2_model_service_delete_model_async.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
Need to exclude these generated samples. This API is a lie and does not really exist.

Reply:
👍  I'll send a CL to disable these snippets for BigQuery.

Reply:
Opened 428009630